examples/contour/01-crds.yaml

```diff
@@ -256,7 +256,8 @@ spec:
                       accessLogLevel:
                         description: "AccessLogLevel sets the verbosity level of the
                           access log. \n Values: `info` (default, meaning all requests
-                          are logged), `error` and `disabled`. \n Other values will
+                          are logged), `error` (meaning all non-success requests are logged),
```

Comment:
Is the description in generated CRDs out of sync with `contourconfig.go` (the source for the yaml files). For example "meaning all 5xx requests are logged" seems missing from `contourconfig.go`?

Reply:
are the `CRD`s manually edited or auto generated?

Reply:
I made them match but of curiosity for next time!

Reply:
You can run `make generate` to generate them from code.

Reply:
smart smart, will do it next time (or if the CI fails because I had a typo)